src/components/InMoreLanguageSwitch.vue

```diff
@@ -0,0 +1,55 @@
+<template>
+	<div class="wikibase-termbox-subsection-switch">
+		<div class="wikibase-termbox-subsection-switch__switch">
+			<a :href="link" @click="clientShowAllLanguages( $event )">{{ message( MESSAGE_KEYS.ALL_LANG ) }}</a>
+		</div>
+	</div>
+</template>
+
+<script lang="ts">
+import { VueConstructor } from 'vue';
+import Component, { mixins } from 'vue-class-component';
+import Messages, { MessagesMixin } from './mixins/Messages';
+
+@Component
+export default class TermBox extends ( mixins( Messages ) as VueConstructor<MessagesMixin> ) {
+	public readonly link: string;
+	constructor() {
+		super();
+		this.link = 'place-holder-link';
+	}
+
+	public clientShowAllLanguages( event: MouseEvent ) {
+		console.log( 'TODO flip all-languages.' );
+		if ( event ) {
+			event.preventDefault();
+		}
+	}
+}
+</script>
+
+<style lang="scss">
+.wikibase-termbox-subsection-switch {
+	& {
+		height: 48px;
+	}
+	
+	.wikibase-termbox-subsection-switch {
+		&__switch {
+			height: 100%;
+			display: flex;
+    		align-items: center;
+			justify-content: center;
+		}
+		
+		&__switch>a {
```

Comment:
Quite a bit of CSS given the story wanted an unstyled link. Is this now per design spec and the respective story (not part of the current sprint) is done through this, or some intermediate step that still requires the other story? If the latter I'm in favor of not having any style at all at this moment.